packages/mui-base/src/SliderUnstyled/SliderUnstyled.tsx

```diff
@@ -278,8 +279,8 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled<
             })}
             style={{
               ...style,
-              pointerEvents: disableSwap && active !== index ? 'none' : undefined,
-              ...thumbProps.style,
```

Comment:
Please don't remove `thumbProps.style`

Reply:
added back, sorry didn't realized it got removed during merging.